config/webpack/webpack.config.js

```diff
@@ -6,6 +6,10 @@ var LodashModuleReplacementPlugin = require("lodash-webpack-plugin");
 
 var SRC = path.resolve("src");
 
+// d3-delaunay and its dependent delaunator need to be transpiled
+var d3Delaunay = path.resolve("../../node_modules/d3-delaunay");
```

Comment:
GLOBAL: These should all be like `require.resolve("d3-delaunay"` as a best practice.